src/index-theme.html

```diff
@@ -0,0 +1,206 @@
+          </code>
+        </pre>
+        <p>
+          Once ECP is rendered, you can easily switch theme without reloading
+          the whole aplpication with the <code>updateTheme</code> function. This
+          is what we do below if the sdk is already initialized (ie the form has
+          already been submitted once).
+        </p>
+        <p>There is 2 types of theme options :</p>
+        <ul>
+          <li>
+            Mandatory :
+            <ul>
+              <li><code>primary</code></li>
+              <li><code>secondary</code></li>
+              <li><code>accent</code></li>
+              <li><code>success</code></li>
+              <li><code>error</code></li>
+              <li><code>background</code></li>
+              <li><code>surface</code></li>
+              <li><code>text</code></li>
+              <li><code>textPrimary</code></li>
+              <li><code>textSecondary</code></li>
+              <li><code>textAccent</code></li>
+              <li><code>textSuccess</code></li>
+              <li><code>textError</code></li>
+            </ul>
+          </li>
+          <li>
+            Optional :
+            <ul>
+              <li><code>textShades</code></li>
+              <li><code>textSurfaceShades</code></li>
+              <li><code>primaryShades</code></li>
+              <li><code>secondaryShades</code></li>
+              <li><code>accentShades</code></li>
+              <li><code>successShades</code></li>
+              <li><code>errorShades</code></li>
+            </ul>
+          </li>
+        </ul>
+        <form id="ecp-form">
+          <div class="form-group">
+            <label for="StreamId">Stream id:</label>
+            <input
+              type="text"
+              class="form-control"
+              id="StreamId"
+              placeholder="Stream ID"
+            />
+          </div>
+          <button type="submit" class="btn">Open ECP</button>
+        </form>
+        <button class="stylebutton" style="display: none;" onclick="changeTheme(lovingTheme)">Lover Mode</button>
+        <button class="stylebutton" style="display: none;" onclick="changeTheme(matrixTheme)">Matrix Mode</button>
+        <button class="stylebutton" style="display: none;" onclick="changeTheme(snowWhiteTheme)">SnowWhite Mode</button>
+        <div class="ecp-frame symphony-ecm" id="symphony-ecm"></div>
+      </div>
+    </div>
+    <script type="text/javascript">
+      const DEFAULT_ORIGIN = "local-dev.symphony.com:9090";
+      const locationUrl = new URL(location.href);
+      const sdkOrigin = locationUrl.searchParams.get("origin");
+      document
+        .getElementById("ecp-form")
+        .addEventListener("submit", (event) => {
+          event.preventDefault();
+          const streamId = document.getElementById("StreamId").value;
+          if (streamId) {
+            if (!window.symphony) {
+              Array.from(document.getElementsByClassName("stylebutton")).forEach((btn) => {
```

Comment:
Shouldn't we show these buttons once ECP is loaded, ie when the render promise resolves